### PR TITLE
Kostas/ops 8656 update setup dd

### PR DESCRIPTION
**Pull request by Kntanos**

### CODE REVIEW

File: app/ssttpdirectdebit/DirectDebitForm.scala [hunk 1]
```diff
@@ -24,31 +24,27 @@ import uk.gov.hmrc.selfservicetimetopay.models.ArrangementDirectDebit.cleanSortC
...
-  def parseIntOption(str: String) = Try(str.toInt).toOption
 
   def condTrue(condition: Boolean, statement: Boolean) = if (condition) statement else true
 
   val directDebitMapping = mapping(
     "accountName" -> text.verifying("ssttp.direct-debit.form.error.accountName.required", _.trim != "")
-      .verifying("ssttp.direct-debit.form.error.accountName.check", x => condTrue(x.trim != "", x.trim.length > 1))
-      .verifying("ssttp.direct-debit.form.error.accountName.check", x => condTrue(x.length > 1, x.matches("^[a-zA-Z].{1,39}$")))
-      .verifying("ssttp.direct-debit.form.error.accountName.not-text", x => condTrue(x.matches("^[a-zA-Z].{1,39}$"), x.length == x.replaceAll("[^a-zA-Z '.& \\/]", "").length)),
+      .verifying("ssttp.direct-debit.form.error.accountName.check", x => condTrue(x.trim != "", x.trim.length <= 70))
```
Comment (darrell-plant) [inline app/ssttpdirectdebit/DirectDebitForm.scala]: `x.trim.length <= 39` makes more sense

File: app/ssttpdirectdebit/DirectDebitForm.scala [hunk 2]
```diff
@@ -24,31 +24,27 @@ import uk.gov.hmrc.selfservicetimetopay.models.ArrangementDirectDebit.cleanSortC
...
+      .verifying("ssttp.direct-debit.form.error.sortCode.not-valid", x => condTrue(x.trim != "", isValidSortCode(x))),
     "accountNumber" -> text.verifying("ssttp.direct-debit.form.error.accountNumber.required", _.trim != "")
-      .verifying("ssttp.direct-debit.form.error.accountNumber.not-valid", x => (x.trim == "") | (validateNumberLength(x, x.length) && validateNumberLength(x, 8)))
+      .verifying("ssttp.direct-debit.form.error.accountNumber.not-valid", x => (x.trim == "") | (isValidNumber(x, x.length) && validateNumberLength(x)))
   )({ case (name, sc, acctNo) => ArrangementDirectDebit(name, cleanSortCode(sc), acctNo) }
     )({ case arrangementDirectDebit => Some((arrangementDirectDebit.accountName, arrangementDirectDebit.sortCode, arrangementDirectDebit.accountNumber)) })
 
   def isValidSortCode(sortCode: String): Boolean = {
-    hasValidSortCodeCharacters(sortCode) && has6Numbers(sortCode)
+    hasValidSortCodeCharacters(sortCode) && hasSixDigits(sortCode)
```
Comment (darrell-plant) [inline app/ssttpdirectdebit/DirectDebitForm.scala]: 👍

File: app/ssttpdirectdebit/DirectDebitForm.scala (hunk 1 above)
Comment (Kntanos) [reply app/ssttpdirectdebit/DirectDebitForm.scala]: oops... yes missed this

File: app/ssttpdirectdebit/DirectDebitForm.scala (hunk 2 above)
Comment (darrell-plant) [reply app/ssttpdirectdebit/DirectDebitForm.scala]: I'm a bit puzzled why there isn't just a simple regex for `sortcode`
something like:
```scala
validSortCodeRegex = """^[0-9]{6}$"""
```

File: app/views/partials/direct_debit_fields.scala.html [hunk 3]
```diff
@@ -14,27 +14,33 @@
...
-@import helpers.forms.textinput
+
 
 @textinput(accountNameField,
-    'label -> Messages("ssttp.direct-debit.form.accountName"),
-    '_maxlength -> "39",
-    '_inputClass -> "input--cleared accountName"
+    'label -> messages("ssttp.direct-debit.form.accountName"),
+    '_labelClass -> "govuk-label--m",
+    '_maxlength -> "70",
```
Comment (darrell-plant) [inline app/views/partials/direct_debit_fields.scala.html]: should be `39`

File: app/ssttpdirectdebit/DirectDebitForm.scala (hunk 2 above)
Comment (darrell-plant) [reply app/ssttpdirectdebit/DirectDebitForm.scala]: and AccountNumber
```code
validSortCodeRegex = """^[0-9]{6, 8}$"""
```

File: app/views/partials/direct_debit_fields.scala.html (hunk 3 above)
Comment (Kntanos) [reply app/views/partials/direct_debit_fields.scala.html]: yes yes...

File: conf/application.conf [hunk 4]
```diff
@@ -207,4 +207,8 @@ contact-frontend.serviceId = "self-service-time-to-pay-frontend"
 
 back-to-tax-account {
     url = "https://www.tax.service.gov.uk/business-account/"
+}
+
+truncateAccName {
+    length = 60
```
Comment (darrell-plant) [inline conf/application.conf]: I don't think this is needed/used

**Review by darrell-plant — DISMISSED** [review]

**Review by darrell-plant — APPROVED** [review]
Bingo!

**Pull request merged**